src/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { BrowserRouter as Router, Routes, Route, Navigate } from "react-router-dom";
+import LogIn from "./pages/LogIn";
+import Sidebar from "./components/sideBar/Sidebar";
+import SignUp from "./pages/SignUp";
+import Home from "./pages/Home";
+import Bikes from "./pages/Bikes";
+import Details from "./pages/Details";
+import Reservations from "./pages/Resevation";
+import "./index.css";
+import AddBike from "./pages/AddBike";
+import DeleteBike from "./pages/DeleteBike";
+import MyReservations from "./pages/MyReservations";
+import sessionStorage from "./helpers/sessions";
+import Error from "./pages/404Page";
+import { useGetBikesQuery } from "./components/api/apiSlice";
+import Spiner from "./reusables/spiner/Spinner";
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this `App` function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, the main `App` function will comply with the latest standards.  Also, don't forget to use ES6 functions for your next projects. 👍 

- [ ] OPTIONAL: Refactor all the other functions to ES6.

I set this to OPTIONAL because there are a lot:

![image](https://github.com/soulemanou-software/car-rental-frontend/assets/57421823/7b64d201-893e-4dfb-8817-9cba13712c12)

Do it if you have the time, or later.

You can do a search in Vscode for the "function" word, and correct as needed.